File: 1_C#(VisualStudio)/1_workspace_ASP.NETCoreMVC/書籍検索アプリ_移植元Java/Models/DatabaseData.cs
using 書籍検索アプリ_移植元Java.Models.beans;
using 書籍検索アプリ_移植元Java.Models.interfaces;

namespace 書籍検索アプリ_移植元Java.Models {
    public class DatabaseData : ILoadData {

        public BookDataBean[] BookDataBeanGet() {

            try {

                return BookDataListDAO.FindAll().ToArray();
            } catch (Exception e) {

                Console.WriteLine(e + "\n例外が発生しましたが、問題なく継続できました。");
                return Array.Empty<BookDataBean>();
            }
        }
    }
}

File: 3_Java(eclipse)/1_workspace_ServletAndJSP/1_書籍検索アプリ/src/main/java/todo/model/DatabaseData.java
package todo.model;

import todo.model.beans.BookDataBean;

public class DatabaseData implements ILoadData{
	
	@Override
	public BookDataBean[] BookDataBeanGet() {
		
		try {
			
			return BookDataListDAO.findAll().toArray(BookDataBean[]::new);
		}catch(Exception e) {
			
			System.out.println("例外が発生しましたが、問題なく継続できました。");
			return new BookDataBean[0];
		}
	}
}